enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,242 @@
+      - Also spreading the backups among all etcd cluster members provide guarantees for disaster recovery in case of losing two members at the same time.
+      - However, using the `local volume` option over all the master nodes will be complicated and error prune.
+      
+
+### Final Storage Design proposal
+Best `Storage` solution is using `hybrid` approach, based on the OCP variant infrastructure.
+
+Based on the `infrastructure` resource type, we can create the storage programmatically according to the underlying infra.
+
+#### Cloud based OCP
+
+    Pros
+    - Utilising `Dynamic provisioning` is best option on cloud based OCP.
+    - Using the default `StorageClass` on the `PVC` manifest will provision a `PV`.
+    - The backups on the `PV` will be always available even if the master node is unavailable.
+    
+    Cons
+    -  There is no possibility to spread the backups over all the master nodes, since the `PV` is accessible within one and only one availability zone. 
+
+
+#### Single-node and Bare metal OCP
+    Only possible option is to use `local volume`. 
+
+### Open Questions
+How to use remote storage for backups on `BM` and `SNO`. 
+
+## Test Plan
+
+An e2e test will be added to practice the scenario as follows 
+
+- Enable AutomatedBackupNoConfig FeatureGate.
+- Verify that a `Etcdbackup` CR has been installed.
+- Verify that a PV, PVC has been created.
+- Verify that a backup has been taken.
+- Verify that the backups are valid according to the retention policy.
+
+## Graduation Criteria
+TBD
+
+### Dev Preview -> Tech Preview
+TBD
+
+### Tech Preview -> GA
+TBD
+
+### Removing a deprecated feature
+TBD
+
+## Upgrade / Downgrade Strategy
+TBD
+
+## Version Skew Strategy
+TBD
+
+## Operational Aspects of API Extensions
+TBD
+
+#### Failure Modes
+TBD
+
+## Support Procedures
+TBD
+
+## Implementation History
+TBD
+
+## Alternatives
+### hostPath
+
+Utilising `hostPath` volumes as a storage solution for the backups.
+
+`hostPath` mounts a path from the node's file system as a volume into the pod.
+
+It supports all OCP variants, including `SNO` and `BM`.  
+
+However, I am strongly against using it for the following reasons
+
+      - `hostPath` could have security impact as it exposes the node's filesystem.
+      - No scheduling guarantees for the pod using `hostpath` as is the case with `localVolume`. The pod could be scheduled on a different node from where the hostPath volume exist.
+      - On the other hand, using `localVolume` and the node affinity within the PV manifest forces the backup pod to be scheduled on a specific node, where the volume is attached.
+      - `localVolume` allows using a separate disk as PV, unlike `hostPath` which mounts a folder from the node's FS.
+      - `localVolume` is handled by the PV controller and the scheduler in different manner, in fact it was created to resolve issues with `hostPath`
```

Comment:
You forgot to add the locality of the backup. Iow. if it happens that backup is kept in the current leader, and that dies, we're loosing that data with it.

Reply:
So the current approach will take backups among all master nodes. Actually we have another issue now, how to distinguish which backup is most up-to-date since we have backups from all master nodes :)